generator/01-header.yml.mustache

```diff
@@ -145,8 +115,11 @@ parameters:
   displayName: Image Stream Namespace
   name: IMAGE_STREAM_NAMESPACE
   value: '{{ Images.ImageStreamNamespace }}'
+- description: Secret to use to encrypt oauth cookies
+  displayName: OAuth Cookie Secret
+  name: OAUTH_COOKIE_SECRET
```

Comment:
For consistencies sake, would it make sense to name this `OPENSHIFT_OAUTH_COOKIE_SECRET` ?

Reply:
It's really Syndesis' oauth cookie rather than OpenShift's. Could rename, but seems OK to me.

Reply:
Fine with that if its intentionally (was a noob remark anyway)